frontend/src/components/templates/ILab/index.less

```diff
@@ -0,0 +1,13 @@
+.pf-v5-c-accordion__expandable-content-body {
+    display: block;
```

Comment:
This is a shared DOM CSS class, so changing its display property impacts other parts of the app as well. It would be better to scope this rule under a parent class or ID that's unique to this component to avoid unintended side effects.

Reply:
Updated the CSS parent's class